replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/legacy/OldSite.java

```diff
@@ -40,5 +40,6 @@ public void fromMap(Map<String, Object> properties) {
 
     setVerifiedUrl(getUrl()); // do this after setUrl()
     setRemoteManaged(false);
+    return CURRENT_VERSION;
```

Comment:
I don't think this should return `CURRENT_VERSION`. It should return maybe `-1` since these sites had no version

Reply:
Changed to 0 since not having a version number was considered version 0